website/docs/r/disk_export.html.markdown

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: "Compute"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_managed_disk_sas_token"
+description: |-
+  Manages a Disk Export.
+---
+
+# azurerm_managed_disk_sas_token
+
+Manages a Disk Export.
+
+Use this resource to obtain a Shared Access Signature (SAS Token) for an existing Managed Disk.
+
+Shared access signatures allow fine-grained, ephemeral access control to various aspects of Managed Disk similar to blob/storage account container.
+
+With the help of this resource, data from the disk can be copied from managed disk to a storage blob or to some other system without the need of azcopy.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "test" {
+  name     = "testrg"
+  location = "West Europe"
+}
+resource "azurerm_managed_disk" "test" {
+  name                 = "tst-disk-export"
+  location             = azurerm_resource_group.test.location
+  resource_group_name  = azurerm_resource_group.test.name
+  storage_account_type = "Standard_LRS"
+  create_option        = "Empty"
+  disk_size_gb         = "1"
+}
+resource "azurerm_managed_disk_sas_token" "test" {
+  managed_disk_id     = azurerm_managed_disk.test.id
+  duration_in_seconds = 300
+  access_level        = "Read"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `managed_disk_id` - (Required) The ID of an existing Managed Disk which should be exported. Changing this forces a new resource to be created.
+
+* `duration_in_seconds` - (Required) The duration for which the export should be allowed. Should be between 30 & 4294967295 seconds.
+
+* `access_level` - (Required) The level of access required on the disk. Supported are Read, Write.
+
+Refer to the [SAS creation reference from Azure](https://docs.microsoft.com/en-us/rest/api/compute/disks/grant-access)
+for additional details on the fields above.
+
+## Attributes Reference
+
+In addition to the Arguments listed above - the following Attributes are exported: 
+
+* `id` - The ID of the Disk Export resource.
+
+* `sas_url` - The computed Shared Access Signature (SAS) of the Managed Disk.
+
+## Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
+
+* `create` - (Defaults to 30 minutes) Used when creating the Disk.
+* `read` - (Defaults to 5 minutes) Used when retrieving the Disk.
+* `delete` - (Defaults to 30 minutes) Used when deleting the Disk.
```

Comment:
we should include how to import the resource here?

Reply:
The SAS tokens are ephemeral, I don't know how useful it would be to import the disk here.